ban/core/versioning.py

```diff
@@ -242,6 +250,7 @@ class Diff(db.Model):
     old = db.ForeignKeyField(Version, null=True)
     # new is empty after delete.
     new = db.ForeignKeyField(Version, null=True)
+    insee = db.CharField(length=5)
```

Comment:
Humm, I feel like I'm missing something, because it seems to me that if I have the PK, I don't need the insee.